Dockerfile

```diff
@@ -113,4 +119,6 @@ RUN if [ "$BUILD_ENV" = "production" ]; then \
     yarn build; \
   fi;
 
-CMD ["yarn", "start"]
+# hadolint ignore=DL3002
+USER root
```

Comment:
This file is serving as dockerfile for both production and dev builds. The things you’re adding shouldn’t be necessary for prod builds. So maybe we should just split into two separate dockerfiles and get rid of the BUILD_ENV stuff?

Reply:
On second thought I'm going to vote to keep a single Dockerfile and use the same config in dev and test and prod for this user stuff. The new implementation is cheap/fast so I think it's OK to use always.